ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -57,6 +53,64 @@ import SwiftUI
     @objc @Published public var layoutType: MSFListCellLayoutType = .automatic
     @objc @Published public var hasDivider: Bool = false
     @objc public var onTapAction: (() -> Void)?
+
+    // Any UIView property will set the value to a type-erased SwiftUI View
```

Comment:
Do you need this comment as you are already outlining this in the main state class documentation?